### ISSUE TITLE
Table sort functionality not accessible to keyboard-only or other AT users. Sort buttons should be included in the tab order and be operable by keyboard

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12126]
Sorting functions such as: Case Details, task(s), Types, Docket Issues, and Days Waiting cannot be accessed with a keyboard. (New: 08/21/2019) You have to be able to do everything with a keyboard to include filter and sort, there is no tab stop on the headers to be able to sort and the filter buttons do nothing when you activate them

**Comment by carodew** [comment]
Linsey sent me the specific WCAG criterion related to this issue: https://www.w3.org/TR/WCAG20-TECHS/F46.html

Reading this, I think we can consider this not a defect. I recommend we close this issue and if the Section 508 Office disagrees we can always reinvestigate.

**Comment by ajspotts** [comment]
This issue actually addresses two separate problems: sort functionality and filter functionality.  I have been able to correct the sort problem with #12898, however the filter problem does not manifest itself during Voiceover testing, only with JAWS.  I've spent a bit of time attempting to track down an appropriate solution but have come up empty handed thus far and am at a stopping point.  

I would suggest either splitting this in to 2 separate issues or accepting the partial sort solution as I've been unable implement a fix for the filter problem at this juncture.

**Comment by lomky** [comment]
No remaining work to be done on this ticket; #12902 will handle the remaining functionality split off. This ticket can either be closed (work complete), or left open to track the larger effort and closed when #12902 is completed.

**Comment by carodew** [comment]
I think if there's no more work on this ticket, my preference would be to close it, since we're tracking the larger effort anyway via #12095. Then I can tell the Section 508 office that this one is complete. :)
